### PR TITLE
Update chaange log for lastest build

### PR DESCRIPTION
**Pull request by dcwebermn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
 
-## [Unreleased]
+See Annotation for Clinical Data [release notes](https://cloud.ibm.com/docs/wh-acd?topic=wh-acd-release-notes) for latest features.
 
-## [0.0.1] - 2019-02-15
+## [Released]
+
+## v2.0 - build v2.0.202106231922
+
+### General Availability
```
Comment (cagrev) [inline CHANGELOG.md:11]: Maybe drop the heading here and just make this text? Would have to see how that looks when formatted that way, and change to "General Availability of IBM Watson Annotator for Clinical Data Container Edition".

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,13 +2,19 @@
 
 All notable changes to this project will be documented in this file.
 
-## [Unreleased]
+See Annotation for Clinical Data [release notes](https://cloud.ibm.com/docs/wh-acd?topic=wh-acd-release-notes) for latest features.
 
-## [0.0.1] - 2019-02-15
+## [Released]
+
+## v2.0 - build v2.0.202106231922
```
Comment (cagrev) [inline CHANGELOG.md:9]: Since "Released" is header 2, should this one be header 3, i.e. ###?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,13 +2,21 @@
...
+
+### General Availability
```
Comment (tpitzen) [inline CHANGELOG.md:11]: Should this say initial release?  I am confused by what this means.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,13 +2,19 @@
...
+
+## v2.0 - build v2.0.202109011707
 
 ### Added
 
-- Added a changelog
+- Refresh of Annotation for Clinical Data images
```
Comment (cagrev) [inline CHANGELOG.md:19]: Refresh of the ACD Service and each of its annotators

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,13 +2,21 @@
...
+See Annotation for Clinical Data [release notes](https://cloud.ibm.com/docs/wh-acd?topic=wh-acd-release-notes) for latest features.
 
-## [0.0.1] - 2019-02-15
+## [Released]
+
+## v2.0 - build v2.0.202106231922
+
+### General Availability
+
+## v2.0 - build v2.0.202109011707
```
Comment (tpitzen) [inline CHANGELOG.md:13]: Should the change list be in reverse order - newest to oldest?

**Review by cagrev — APPROVED** [review]
Approved pending changes per individual comments. Let's use that as a starting point and we can continue to refine from there if others have additional input.

**Pull request merged**